src/components/report-issue.tsx

```diff
@@ -38,15 +39,20 @@ export function ReportIssueForm({
         issues += (checks[i] as HTMLInputElement).value + "\n";
       }
     }
-    issues += (
+    const reportContent = (
       (event.target as HTMLFormElement).querySelector(
         "#reportContent",
       ) as HTMLInputElement
     ).value;
```

Comment:
Is there a reason why you are storing and accessing this information via DOM as opposed to storing it and accessing it via useState?